lib/web_ui/test/engine/semantics/text_field_test.dart

```diff
@@ -63,6 +63,29 @@ void testMain() {
     semantics().semanticsEnabled = false;
   });
 
+  test('tap detection works', () async {
+    debugBrowserEngineOverride = BrowserEngine.webkit;
+    debugOperatingSystemOverride = OperatingSystem.iOs;
```

Comment:
These need to be set back to `null` at the end of the test.

Reply:
Done!

Reply:
nit: resetting using [addTearDown](https://pub.dev/documentation/test_api/latest/test_api.scaffolding/addTearDown.html) would ensure we reset them even when the test fails. This way, this global change won't carry over to other tests.